plugins/glassfish/src/net/java/visualvm/modules/glassfish/dataview/Bundle.properties

```diff
@@ -2,13 +2,13 @@
 WebModuleViewPanel.jPanel1.border.title=Sessions
 WebModuleViewPanel.labelCurrent.text=Active Current
 WebModuleViewPanel.labelMax.text=Active Max
-WebModuleViewPanel.labelPassivated.text=Passivated
+WebModuleViewPanel.labelZ_ThisIsNotAWordAndWasNotUsedWhenItWasOpenInitiallyReleased.text=Z_ThisIsNotAWordAndWasNotUsedWhenItWasOpenInitiallyReleased
```

Comment:
This commit is unlike the rest of the commits. I have no idea what `Passivated` means. It also appears unused.